pandas/core/indexes/base.py

```diff
@@ -2536,12 +2537,17 @@ def asof(self, label):
 
     def asof_locs(self, where, mask):
         """
-        where : array of timestamps
-        mask : array of booleans where data is not NA
+        Parameters
```

Comment:
If you know what this function does, can you add a short summary?

Reply:
@datapythonista I've added in a summary based on what I've understood from playing around with the function. I think you / the original author of the function should verify it though.

Reply:
@jreback, do you mind reviewing the summary?